.github/workflows/trigger-rebuild-repo-package.yml

```diff
@@ -0,0 +1,33 @@
+name: Trigger to build a deb package from repo
+
+on:
+  pull_request:
+    types:
+      - closed
+    branches:
+      - current
+  workflow_dispatch:
+
+jobs:
+  trigger-build:
+    if: github.event.pull_request.merged == true || github.event_name == 'workflow_dispatch'
+    runs-on: ubuntu-latest
+
+    env:
+      REF: main  # Used for curl to trigger build package
+
+    steps:
+      - name: Set variables
+        run: |
+          echo "PACKAGE_NAME=$(basename ${{ github.repository }})" >> $GITHUB_ENV
+
+      - name: Trigger rebuild for ${{ env.PACKAGE_NAME }}
+        run: |
+          set -eux
```

Comment:
Do we really want to expose all debug output?

Reply:
Fixed!